CRM/CiviAwards/Service/AwardProfile.php

```diff
@@ -53,7 +63,25 @@ public function createProfileFields(array $customFields, $profileId) {
     $this->addContactIdField($profileId);
   }
 
-    $this->addContactIdField($profileId);
+  /**
+   * Validates review_field options to ensure legal options are passed.
+   *
+   * @param array
```

Comment:
No parameter name?
Did you run the git pre-commit hook on local before committing?